cric-predict/src/Components/CreateGroup.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import '../Stylesheets/App.css';
+import axios from 'axios';
+import {Button, Modal, Form, Row, Col}  from 'react-bootstrap';
+
+
+export default class CreateGroupsComponent extends React.Component  {
+
+        constructor(){
+            super();
+            this.state = {
+                name: '',
+                privacy: 'public',
+                all_users:[],
+                selected_users: [],
+                filtered_users:[],
+                users_copy: [],
+                showHide : false
+            }
+        }
+
+        async componentDidMount() {
+            const response = await axios.get('http://localhost:8000/api/user/list/',{
+            headers: {
+                'Authorization': "Bearer "+window.localStorage.getItem('access_token')
+            }
+            })
+            const users_list = response.data
+            this.setState({all_users: users_list, filtered_users:users_list, users_copy: users_list})
```

Comment:
why do we need to make a copy of users? if it's necessary we should have a better name for it and use the spread operator to make a copy of an array (preferred way to make copies of reference types in ES6)